addons/arrays/test_shuffle.sqf

```diff
@@ -21,18 +21,14 @@ TEST_OP(count _result,==,count _original,_fn);
     TEST_OP(_x,in,_original,_fn);
 } forEach _result;
 
-// Test depecated version.
+// test modify original
```

Comment:
Look at the last unit test you altered (line 31). That one is refactorised for the new parameter format. The first two tests in the original version should be kept, but updated in the same way. Then add the new test you added for the "modify input array" option.